views/movies/movies.hbs

```diff
@@ -0,0 +1,11 @@
+<h2>All Movies</h2>
+{{#each movies}}
+<a href="movies/{{_id}}/details"><h3>{{title}}</h3></a>
+<h4>{{genre}}</h4>
+<p>{{cast}}</p>
```

Comment:
Same in here, if you want to show the name of the cast for example, you would need to populate and then pass it here like before.

```
{{#each cast}}
<p>{{name}}</p>
{{/each}}
```